books/RayTracingTheNextWeek.html

```diff
@@ -2256,7 +2256,319 @@
 
 
 
-Rectangles and Lights
+Quadrilaterals
+====================================================================================================
+We've managed to get more than half way through this three-book series using spheres as our only
+geometric primitive. Time to add our second primitive: the quadrilateral.
+
+
+Ray-Plane Intersection
+-----------------------
+Spheres are generally the first ray tracing primitive because their implicit formula makes it so
+easy to solve for ray intersection. Like spheres, planes also have an implicit formula, and
+ray-plane intersection is even _easier_ to solve.
```

Comment:
Like spheres, planes also have an implicit formula, and we can use their implicit formula to produce an algorithm that solves for ray-plane intersection. Indeed, ray-plane intersection is even _easier_ to solve.

Reply:
Something like that. It might be good to reinforce the knowledge that we use the implicit formula to create an algorithm for the intersection, since working from implicit formula to intersection algorithm is used all over the place in 3D math.

Reply:
Done.